src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtSpoeAreasOfLawControllerTest.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.hmcts.dts.fact.controllers.admin;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.services.admin.AdminCourtSpoeAreasOfLawService;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.dts.fact.util.TestHelper.getResourceAsJson;
+
+@WebMvcTest(AdminCourtSpoeAreasOfLawController.class)
+@AutoConfigureMockMvc(addFilters = false)
+public class AdminCourtSpoeAreasOfLawControllerTest {
+    private static final String BASE_PATH = "/admin/courts/";
+    private static final String CHILD_PATH = "/SpoeAreasOfLaw";
+    private static final String TEST_SLUG = "unknownSlug";
+    private static final String TEST_COURT_AREAS_OF_LAW_PATH = "court-spoe-areas-of-law.json";
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Autowired
+    private transient MockMvc mockMvc;
+
+    @MockBean
+    private AdminCourtSpoeAreasOfLawService adminService;
+
+
+    @Test
+    void shouldReturnAllSpoeAreasOfLaw() throws Exception {
+        final String expectedJson = getResourceAsJson(TEST_COURT_AREAS_OF_LAW_PATH);
+        final List<AreaOfLaw> spoeAreasOfLaw = asList(OBJECT_MAPPER.readValue(expectedJson, AreaOfLaw[].class));
+
+        when(adminService.getAllSpoeAreasOfLaw()).thenReturn(spoeAreasOfLaw);
+
+        mockMvc.perform(get(BASE_PATH + CHILD_PATH))
+            .andExpect(status().isOk())
+            .andExpect(content().json(expectedJson));
+    }
+
+
+
+    @Test
+    void shouldReturnCourSpoeAreasOfLaw() throws Exception {
```

Comment:
only worth fixing if you're in the code, dont bother if not, just a small typo

Reply:
corrected